queries/powershell/highlights.scm

```diff
@@ -0,0 +1,323 @@
+    "Object" "System.Object" "ValueType" "System.ValueType" "Hashtable"
+    "System.Collections.Hashtable" "Collections.Stack" "System.Collections.Generic.Stack"
+    "Collections.Dictionary" "System.Collections.Dictionary" "Collections.DictionaryEntry"
+    "System.Collections.DictionaryEntry" "Collections.ArrayList" "System.Collections.ArrayList"
+    "Management.Automation.SwitchParameter" "System.Management.Automation.SwitchParameter"
+    "Management.Automation.ActionPreference" "System.Management.Automation.ActionPreference"
+    "Management.Automation.ConfirmImpact" "System.Management.Automation.ConfirmImpact"
+    "Management.Automation.CommandInfo" "System.Management.Automation.CommandInfo"
+    "Management.Automation.ProviderInfo" "System.Management.Automation.ProviderInfo"
+    "Management.Automation.PSDriveInfo" "System.Management.Automation.PSDriveInfo"
+    "Management.Automation.PSVariable" "System.Management.Automation.PSVariable"
+    "Management.Automation.AliasInfo" "System.Management.Automation.AliasInfo"
+    "Management.Automation.PathInfo" "System.Management.Automation.PathInfo"
+    "Management.Automation.PathInfoStack" "System.Management.Automation.PathInfoStack"
+    "Management.Automation.ApplicationInfo" "System.Management.Automation.ApplicationInfo"
+    "Management.Automation.CmdletInfo" "System.Management.Automation.CmdletInfo"
+    "Management.Automation.ExternalScriptInfo" "System.Management.Automation.ExternalScriptInfo"
+    "Management.Automation.FunctionInfo" "System.Management.Automation.FunctionInfo"
+    "Management.Automation.CommandTypes" "System.Management.CommandTypes"
+    "Management.Automation.ScopedItemOptions" "System.Management.Automation.ScopedItemOptions"
+    "Management.Automation.PSTypeName" "System.Management.Automation.PSTypeName"
+    "Management.Automation.ParameterMetadata" "System.Management.Automation.ParameterMetadata"
+    "Management.Automation.CommandParameterSetInfo"
+    "System.Management.Automation.CommandParameterSetInfo"
+    "Management.Automation.SessionStateEntryVisibility"
+    "System.Management.Automation.SessionStateEntryVisibility" "Management.Automation.FilterInfo"
+    "System.Management.Automation.FilterInfo" "Management.Automation.ModuleType"
+    "System.Management.Automation.ModuleType" "Management.Automation.PSModuleInfo"
+    "System.Management.Automation.PSModuleInfo" "Management.Automation.PSCustomObject"
+    "System.Management.Automation.PSCustomObject" "Management.Automation.PSScriptCmdlet"
+    "System.Management.Automation.PSScriptCmdlet" "Management.Automation.ErrorRecord"
+    "System.Management.Automation.ErrorRecord" "Management.Automation.PSCredential"
+    "System.Management.Automation.PSCredential" "Management.Automation.PSMethod"
+    "System.Management.Automation.PSMethod")
+  )
+
+; Function definitions
+;---------------------
+(function_statement
+  (function_name) @function)
+
+; Classes, fields and methods definitions
```

Comment:
Maybe also `@property` for `(key_expression)`

Reply:
Missed that, thanks!